examples/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+# Minimum CMake required

```

Comment:
This PR has both cmake/examples.cmake and examples/CMakeLists.txt. Are we going to keep both or just one of them?